ctsit/PyUtils
# Add `update_model` and return created objects on insert/update

**What does this change do?** _Please be clear and concise._

This PR adds the ability to update a model. As well as return the created/updated objects on `insert_data`/`update_model`.

**Why was this change made?** _Including an issue number is sufficient. Otherwise, briefly explain the benefit of the change._

This change was made to be able to update existing entries in the database, and retrieve inserted/updated models without needing to perform an additional action.

## Verification

_List the steps needed to make sure this thing works. Be precise._

1. Verify that the login in `test_update_model` is sound.
2. Verify that all workflows pass

## Affirmations

All of these should have a check by them. Any exception requires an explanation.
